compile-css.js

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
maybe we should copy the devsummit site and put build related tasks into a `lib/` directory in the root.

As a follow up, I want to rename the various `lib/` directories we have in `src/` because I think they're a bit confusing at the moment.

Reply:
Sure, but this PR doesn't change the status quo now. I probably agree but I wonder how big it will get.